src/main/scala/dpla/ingestion3/enrichments/ParseDateEnrichment.scala

```diff
@@ -35,11 +38,20 @@ class ParseDateEnrichment {
     //this iterates over the list, executes the next function, and returns it if there's a result.
     //otherwise, it continues until the end and returns None.
     //I wasn't able to find a better way to emulate ||= from Ruby. There's probably a better way.
```

Comment:
Off the top of my head, I can't come up with a better way.  This way makes sense to me.